Calibration/HcalCalibAlgos/test/HcalIsoTrkAnalyzer.cc

```diff
@@ -992,13 +996,12 @@ double HcalIsoTrkAnalyzer::trackP(const reco::Track* pTrack,
 
   double pmom = -1.0;
   if (genParticles.isValid()) {
-    reco::GenParticleCollection::const_iterator p;
     double mindR(999.9);
-    for (p=genParticles->begin(); p!=genParticles->end(); ++p) {
+    for (const auto &p : (reco::GenParticleCollection)(*genParticles)) {
```

Comment:
remove the (reco::GenParticleCollection)

Reply:
done